Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4260,18 +4263,18 @@ def is_valid_type(_type):
                 )
 
         def __init__(
-            self,
-            algorithm,  # type: str
-            length,  # type: int
-            publickey=None,  # type: str
-            p=None,  # type: str
-            q=None,  # type: str
-            g=None,  # type: str
-            modulus=None,  # type: str
-            exponent=None,  # type: int
-            x=None,  # type: str
-            y=None,  # type: str
-            curve=None  # type: str
+                self,
+                algorithm,  # type: str
```

Comment:
Revert all these indentations. I don't think it was intended

Reply:
Done